app/src/swap.h

```diff
@@ -0,0 +1,93 @@
+/*******************************************************************************
+*   (c) 2016 Ledger
+*   (c) 2018 -2022 Zondax AG
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+********************************************************************************/
+#pragma once
+
+#include "stdbool.h"
+#include "stdint.h"
+#include "parser_common.h"
+
+#define RUN_APPLICATION         1
+#define SIGN_TRANSACTION        2
+#define CHECK_ADDRESS           3
+#define GET_PRINTABLE_AMOUNT    4
```

Comment:
This names look really common, and probably not really semantic. Why not using an `enum` here?